FWCore/Framework/test/stubs/TestOneFilters.cc

```diff
@@ -228,6 +229,191 @@ namespace edmtest {
       }
     };
 
+    class ProcessBlockIntFilter : public edm::one::EDFilter<edm::ProcessBlockCache<fltr::Cache>> {
```

Comment:
How does the ProcessBlock cache get created? There doesn't appear to be a mechanism in this module. If a ::one module has a RunCache the beginRun function returns a `std::shared_ptr` to the cache object.

Reply:
I just noticed that this question applies to every single module type which can declare a `edm::ProcessBlockCache`.

Reply:
I planned to address this in one of the future PR's implementing persistence. There are no tests of this yet and the function creating the caches will never actually be called by any code in this PR. But the interface is in this PR, for example for global modules in FWCore/Framework/interface/global/implementors.h:
```
virtual std::shared_ptr<C> accessInputProcessBlock(ProcessBlock const&) const { return std::shared_ptr<C>(); }
```